### PR TITLE
doc: add meeting minutes for 28th October

### PR DESCRIPTION
**Pull request by gireeshpunathil**
Fixes: https://github.com/nodejs/diagnostics/issues/438

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by Qard — APPROVED** [review]

File: wg-meetings/2020-10-28.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+* Michael Dawson (@mhdawson)
+* Chengzhong Wu (@legendecas) 
+* Gireesh Punathil (@gireeshpunathil)
+
+
+## Agenda
+
+## Announcements
+ 
+*Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
```
Comment (ExE-Boss) [inline wg-meetings/2020-10-28.md:23]: This currently produces a broken result:

> *Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

---

It should be:
```suggestion
Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
```
> Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by gireeshpunathil** [comment]
landed in 75d0f74bcc89e9ab537af72b03129775f641f2ce

**Pull request merged**